### ISSUE TITLE
CAN-Examples fail to build

### ISSUE THREAD

**Issue by Jueff** [Hardi-St/MobaLedLib#14]
the build of the examples 
- 11.CAN_Bus_MS2_RailwaySignal
- 12.CAN_Bus_MS2_Switch_Lights
- 20.Huge_Configuration

fails with error

```
...\arduino\build\libraries\MCP_CAN_lib\mcp_can.cpp.o (symbol from plugin): In function `MCP_CAN::mcp2515_reset()':
(.text+0x0): **multiple definition of `MCP_CAN::mcp2515_reset()**'
...\arduino\build\libraries\MobaLedLib\mcp_can_nd.cpp.o (symbol from plugin):(.text+0x0): first defined here
...\arduino\build\libraries\MCP_CAN_lib\mcp_can.cpp.o (symbol from plugin): In function `MCP_CAN::mcp2515_reset()':
(.text+0x0): **multiple definition of `MCP_CAN::mcp2515_readRegister**(unsigned char)'

```

**Comment by Jueff** [comment]
@Hardi-St: could you please have a look on that issue?

**Comment by Jueff** [comment]
fixed with commit 49f6b09  and 3b64c26
